src/contracts/libraries/MiddlewareUtils.sol

```diff
@@ -3,20 +3,20 @@
 pragma solidity =0.8.12;
 
 /**
- * @title Library of functions shared across DataLayr.
+ * @title Library of functions shared across Middlewares.
  * @author Layr Labs, Inc.
  * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
  */
 library MiddlewareUtils {
+    uint256 internal constant MAX_QUORUM_BITMAP = type(uint192).max;
+
     /// @notice Finds the `signatoryRecordHash`, used for fraudproofs.
     function computeSignatoryRecordHash(
-        uint32 globalDataStoreId,
-        bytes32[] memory nonSignerPubkeyHashes,
-        uint256 signedStakeFirstQuorum,
-        uint256 signedStakeSecondQuorum
+        uint32 referenceBlockNumber,
+        bytes32[] memory nonSignerPubkeyHashes
     ) internal pure returns (bytes32) {
         return keccak256(
-            abi.encodePacked(globalDataStoreId, nonSignerPubkeyHashes, signedStakeFirstQuorum, signedStakeSecondQuorum)
+            abi.encodePacked(referenceBlockNumber, nonSignerPubkeyHashes)
         );
     }
 }
```

Comment:
big question:
*What is this library actually supposed to be?*
Feels like it contains a single constant that belongs probably in the VoteWeigher, and a function that belongs in the BLSSignatureChecker.
I recommend moving these functions to their proper locations and deleting this library.